libs/sdk-ui-filters/src/DateFilter/DateFilterBody/DateFilterHeader.tsx

```diff
@@ -1,13 +1,20 @@
-// (C) 2019 GoodData Corporation
-import React from "react";
+// (C) 2019-2025 GoodData Corporation
+import React, { useEffect, useRef } from "react";
 import { DateFilterRoute } from "./types.js";
 
 export const DateFilterHeader: React.FC<{
     children: any;
     changeRoute: (route: DateFilterRoute) => void;
 }> = ({ children, changeRoute, ...otherProps }) => {
+    const buttonRef = useRef<HTMLButtonElement>(null);
+
+    useEffect(() => {
```

Comment:
consider reusing useAutofocusOnMount instead of this

Reply:
Changed to `useAutofocusOnMount`